lib/models/user_model.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:badiup/models/user_setting_model.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+class User {
+  final String email;
+  final String name;
+  final RoleType role;
+  final DocumentReference setting;
```

Comment:
Why is setting stored as a DocumentReference but Address is stored as is in the document itself (and not as a reference)?

Reply:
I will store customer's addresses as a List<DocumentReference>.